packages/storybook/.storybook/main.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
-  stories: [
-    '../../core/src/layout/**/*.stories.tsx',
-    '../../core/src/components/**/*.stories.tsx',
-  ],
+  stories: ['../../../**/*.stories.tsx'],
```

Comment:
Not sure we want this tbh, this storybook is really for plugin development and what is available in core. Should def be `../../core/**` tho. Perhaps a separate storybook for the plugin if you want one?

Reply:
yeah, it seems like it would be very helpful as part of the overall Backstage CLI as a thing that plugins can get out of the box. It's pretty typical to develop un-connected components in their many states inside a storybook and then hook them up in the main application, confident that you've covered things.

Reply:
for now I'll dump the stories and save it for another time.